contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Before=origin-master.service
```

Comment:
I'm just going to drop it, it's in the node service anyway and if we were to make the openvswitch image generic it wouldn't make sense.